doc/doxygen/src/Examples.dox

```diff
@@ -236,6 +264,10 @@
  * if TextEncodingTrait.isImbuedTo(policy):
  *   print("The manager can provide the text encoding used")
  *
+ * resolved_asset = manager.resolve(
+ *         entity_reference,
+ *         {LocatableContentTrait.kId, TextEncodingTrait.kId},
+ *         ResolveAccess.kRead, context)
```

Comment:
The reason we added this is to provide a motivation to check capability, so we can showcase both management policy + capability in the same example. Taking this out would imply adding a second example just for capability so we can showcase them seperately, you think that's the thing to do?

Reply:
Yeah the problem we faced was choosing a capability to add to the example, then not actually using the functionality we tested for.  No big deal to remove it though.

Reply:
The sentence above could be tweaked a ilttle to say something like:

"This example demonstrates how to query a manager in relation to a specific entity type - in this case a simple text file, and inspect its capabilities and the data it may be able to resolve for it"

Reply:
Aye https://github.com/OpenAssetIO/OpenAssetIO/pull/1141/commits/db699306fe3528726c8da01f8a23385fbfeecfaf